### ISSUE TITLE
lora key not loaded

### ISSUE THREAD

**Issue by yanliu-github** [kijai/ComfyUI-HunyuanVideoWrapper#234]
System Information:
OS: POSIX
Python Version: 3.10.12 (main, Jul 5 2023, 18:54:27) [GCC 11.2.0]
Embedded Python: false
PyTorch Version: 2.5.1+cu124
RAM:
Total RAM: 440.71 GB
Free RAM: 52.91 GB
Device:
Name: NVIDIA GeForce RTX 4090 (cuda:0) - Supports cudaMallocAsync
Type: CUDA
VRAM Total: 23.64 GB
VRAM Free: 5.48 GB
Torch VRAM:
Total: 19.25 GB
Free: 1.6 GB

log:

got prompt
llm prompt attention_mask shape: torch.Size([1, 161]), masked tokens: 14
clipL prompt attention_mask shape: torch.Size([1, 77]), masked tokens: 17
model_type FLOW
Scheduler config: FrozenDict([('num_train_timesteps', 1000), ('shift', 9.0), ('reverse', True), ('solver', 'euler'), ('n_tokens', None), ('_use_default_values', ['num_train_timesteps', 'n_tokens'])])
Using accelerate to load and assign model weights to device...
Loading LoRA: hyvideo_FastVideo_LoRA-fp8 with strength: 1.0
lora key not loaded: diffusion_model.double_blocks.0.img_attn.proj.diff_b
lora key not loaded: diffusion_model.double_blocks.0.img_attn.proj.lora_down.weight
lora key not loaded: diffusion_model.double_blocks.0.img_attn.proj.lora_up.weight
lora key not loaded: diffusion_model.double_blocks.0.img_attn.qkv.diff_b
lora key not loaded: diffusion_model.double_blocks.0.img_attn.qkv.lora_down.weight
lora key not loaded: diffusion_model.double_blocks.0.img_attn.qkv.lora_up.weight
lora key not loaded: diffusion_model.double_blocks.0.img_mlp.0.diff_b
lora key not loaded: diffusion_model.double_blocks.0.img_mlp.0.lora_down.weight
lora key not loaded: diffusion_model.double_blocks.0.img_mlp.0.lora_up.weight
lora key not loaded: diffusion_model.double_blocks.0.img_mlp.2.diff_b
lora key not loaded: diffusion_model.double_blocks.0.img_mlp.2.lora_down.weight
lora key not loaded: diffusion_model.double_blocks.0.img_mlp.2.lora_up.weight
lora key not loaded: diffusion_model.double_blocks.0.img_mod.lin.diff_b
lora key not loaded: diffusion_model.double_blocks.0.img_mod.lin.lora_down.weight
lora key not loaded: diffusion_model.double_blocks.0.img_mod.lin.lora_up.weight
..........

workflow image:
![image](https://github.com/user-attachments/assets/a48ab54c-ae36-48a0-9b70-c0d2def583cc)

What is the issue? How should it be resolved?

**Comment by Fredd4e** [comment]
I get the same issue, not sure what it means

**Comment by kijai** [comment]
The FastVideo LoRA was extracted using the core ComfyUI LoRA extraction node, so it doesn't work with this wrapper currently. It's slightly worse than using the full FastVideo model though, so that's still preferred way to use it. I extracted it as LoRA mostly to test if adjusting the strength is useful, which it can be...but I don't yet have any conclusive results about that.

**Comment by yanliu-github** [comment]
> The FastVideo LoRA was extracted using the core ComfyUI LoRA extraction node, so it doesn't work with this wrapper currently. It's slightly worse than using the full FastVideo model though, so that's still preferred way to use it. I extracted it as LoRA mostly to test if adjusting the strength is useful, which it can be...but I don't yet have any conclusive results about that.

Okay, I think this is a great project. Will you consider supporting this LoRA in the future?

**Comment by lucobit** [comment]
Maybe Convert LoRA to Other Formats with AI Software Musubi Tuner Gui code by Kohya Gui by TTP , such as converted.safetensors format can fix the issue

**Comment by hsume2** [comment]
I've been experiencing the same thing and converting the LoRa using https://github.com/kohya-ss/musubi-tuner/blob/main/convert_lora.py seems to have done the trick.

```
python conver_lora.py --input <IN> --output <OUT> --target other
```

**Comment by 4lt3r3go** [comment]
> I've been experiencing the same thing and converting the LoRa using https://github.com/kohya-ss/musubi-tuner/blob/main/convert_lora.py seems to have done the trick.
> 
> ```
> python conver_lora.py --input <IN> --output <OUT> --target other
> ```



OH YES i wish that fast lora working on Kijai's.. since is better to use the full model with the lora loaded in.
Using the fast model sometimes introduce glitches, but using full model+lora the glitches amount are less present or more controllable.
I have no idea how to convert it otherwise was the top in my priority list, instead of doing crazy spaghetti workflows for the community 😁

**Comment by sayakpaul** [comment]
Maybe we could try to get the LoRA converted to the diffusers format?
